recipes/libtool/libtool-2.4.2.inc

```diff
@@ -0,0 +1,14 @@
+LICENSE = "GPL-2.0+"
+LICENSE_${PN}-libltdl = "LGPL-2.1+"
+
+require ${PN}.inc
+
+SRC_URI:>native = " file://syslib.patch"
+SRC_URI:>cross = " file://syslib.patch"
+SRC_URI:>sdk-cross = " file://syslib.patch"
+SRC_URI:>cross = "file://cross_compile.patch"
```

Comment:
Indeed ... not discover cause that multiple appends on same flag and variable are not handled (use only the last one) which is on this case SRC_URI:>cross = " ${SRC_URI_WINDOWS}"

Btw the missing  patches break some x11 libs compilation